liquibase-core/src/main/java/liquibase/parser/core/formattedsql/FormattedSqlChangeLogParser.java

```diff
@@ -26,6 +26,155 @@
 public class FormattedSqlChangeLogParser implements ChangeLogParser {
 
 
+    private static final String FIRST_LINE_REGEX = "\\-\\-\\s*liquibase formatted.*";
```

Comment:
The standard for using the word changeset is one word, not two (change set). The Strings and Pattern names that have `*CHANGE_SET*` should be `*CHANGESET*` for consistency.

Reply:
"Changeset" is one word so it shouldn't be split with the `_` in the variable names. Does that make more sense, @arturobernalg? Maybe? :)

Reply:
Sorry @XDelphiGrl, is this not the regex to match the first line of a changelog in sql format?

`-- liquibase formatted sql`

Reply:
That is correct, @MalloD12. That is the correct regex for the starting line of a formatted SQL changelog.

I think part of the confusion is I added my comment on a line where it doesn't make sense. I put it on the first line in the file because it applies to multiple lines BUT I can see where this is confusing. The line for an example is Line 39 (CHANGE_SET_REGEX ... I suggest we change it to CHANGESET_REGEX). 

My issue is just with the names of the variables being "CHANGE_SET" instead of "CHANGESET". We've done some cleanup lately to be consistent:
[https://github.com/liquibase/liquibase/pull/3042](https://github.com/liquibase/liquibase/pull/3042).

Reply:
After discussion with @nvoxland and @filipelautert, we're going to stay with the `CHANGE_SET` and not make a change to `CHANGESET`. Resolving this conversation.